src/recipe/Baked.Recipe.Service.Application/Theme/Admin/ErrorHandlingOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Baked.Ui;
+
+namespace Baked.Theme.Admin;
+
+public class ErrorHandlingOptions : INamedSettings
+{
+    public DefaultHandlerOptions DefaultHandler { get; init; } = new();
```

Comment:
why does this have one extra nested class, just place `List<HandlerOption>` here, and that's enough?